README.md

```diff
@@ -19,10 +19,10 @@ The goal of this specification is to enable the creation of interoperable tools
 
 ## Overview
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
-At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md) and may with a [manifest list](manifest-list.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
+At a high level the manifest list points to specific image manifests for one or more platform and the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
 The image configuration includes information such as application arguments, environments, etc.
-The combination of the image manifest, image configuration, and one or more filesystem layers is called the "OCI Image".
+The combination of the image manifest list, image manifest, image configuration, and one or more filesystem layers is called the "OCI Image".
```

Comment:
I've also consolidated this with the earlier OCI Image sentence in #465.  I'm happy to rebase that consoliation if this PR lands first.